dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -26,6 +26,7 @@
 //= require md5
 
 // Application assets
+//= require namespace
```

Comment:
is this here to make sure it comes before the other .js files included by the require_directory line below? if so maybe add a comment so nobody tries to remove the "redundant" require